integrations/editor_test.go

```diff
@@ -104,3 +106,55 @@ func TestCreateFileOnProtectedBranch(t *testing.T) {
 	// Check body for error message
 	assert.Contains(t, string(resp.Body), "Can not commit to protected branch &#39;master&#39;.")
 }
+
+func testEditFile(t *testing.T, session *TestSession, urlstr string) {
```

Comment:
I thought of that at first but another side of me told that function parameters should no more than 4 if possible otherwise it would look awkward. I will reconsider your advice or if anybody else there is willing to chime in? I don't strongly insist on either way but I'd like to have more confirmations.